app/assets/javascripts/effective_datatables/bulk_actions.js.coffee

```diff
@@ -86,6 +88,11 @@ $(document).on 'click', '.dataTables_wrapper .buttons-bulk-actions a', (event) -
         $table.one 'draw.dt', (e) -> $(e.target).DataTable().flash(error, 'danger')
         $table.DataTable().draw()
     )
+  else if redirect_mode
```

Comment:
Instead of submitting a form, could we use the $.ajax, as just below, to perform this action?

Cheers.